channels/migrations/0001_initial.py

```diff
@@ -110,7 +110,7 @@ class Migration(migrations.Migration):
                     "field",
                     models.ForeignKey(
                         on_delete=django.db.models.deletion.CASCADE,
-                        to="channels_fields.FieldChannel",
```

Comment:
Since this is essentially a new app and any pre-existing `FieldChannel` objects won't be preserved, Instead of copying all these migrations over and adjusting them, could just start with a clean slate:
- Run `./manage.py migrate channels zero` to clean out your local database tables for this app.
- Remove migrations folder altogether
- Run `./manage.py makemigrations channels`
- End up with just one migration file (`0001_initial.py`) that is up to date.